ldap/servers/slapd/back-ldbm/db-bdb/bdb_import.c

```diff
@@ -2595,10 +2595,18 @@ bdb_public_bdb_import_main(void *arg)
             slapi_task_finish(job->task, ret);
         }
     } else {
+        dblayer_private *priv = NULL;
```

Comment:
Something not clear to me is that this part of code is in the import 'error' handling. Should not it be done only after the creation of the 'ancestorid' index ?